lib/urlwatch/filters.py

```diff
@@ -130,6 +132,9 @@ def __init__(self, job, state):
         super().__init__(job, state)
 
         self.hooks = None
+        if not os.path.exists(self.FILENAME):
+            self.FILENAME = AppDirs('urlwatch').user_config_dir + '/lib/hooks.py'
```

Comment:
Use `os.path.join()` instead of hardcoding `/` as path separator.

Reply:
Fixed